### ISSUE TITLE
False Positive --virtual

### ISSUE THREAD

**Issue by Vad1mo** [replicatedhq/dockerfilelint#79]
```
FROM ruby:2.3-alpine
MAINTAINER 8gears AG <hello@8gears.com>

WORKDIR /
COPY . /

EXPOSE 80 443

RUN apk --no-cache add -q -U --virtual build-dependencies \
    gcc g++ make musl-dev openssl-dev libffi-dev curl-dev mariadb-dev libxml2-dev zlib-dev libgcrypt-dev libxslt-dev coreutils\
    && apk --no-cache add bash zlib libcurl libxslt tzdata mariadb-libs mariadb-client-libs nodejs\
    && echo 'gem: --verbose --no-document' > /etc/gemrc \
    && gem update --no-document --quiet --system \
    && bundle config build.nokogiri --use-system-libraries \
    && bundle install --jobs=`nproc` --retry=3 --no-cache --clean \
    && rake assets:precompile && rake webpack:compile \
    && apk del --purge build-dependencies ; bash -c "rm -rf /{tmp,root}/{*,.??*}" \
    && rm -rf /usr/lib/ruby/gems/*/cache/* /var/cache/apk/*

COPY . /
ADD . . 

CMD ["/start.sh"]


```

It says I should use  `--virtual`however  I do use it but in the second apk statement the first one is fine as is.
